Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -211,105 +216,120 @@ Configure the network that will be enabled for the cluster and services.
     Changing networking configuration after a Ceph cluster has been deployed is NOT
     supported and will result in a non-functioning cluster.
 
-#### Host Networking
+#### Ceph public and cluster networks
 
-To use host networking, set `provider: host`.
+Ceph daemons can operate on up to two distinct networks: public, and cluster.
 
-If the host networking setting is changed in a cluster where mons are already running, the existing mons will
-remain running with the same network settings with which they were created. To complete the conversion
-to or from host networking after you update this setting, you will need to
-[failover the mons](../../Storage-Configuration/Advanced/ceph-mon-health.md#failing-over-a-monitor)
-in order to have mons on the desired network configuration.
+Ceph daemons always use the public network, which is the Kubernetes pod network by default. The
+public network is used for client communications with the Ceph cluster (reads/writes).
 
-#### Multus
-
-Rook supports addition of public and cluster network for ceph using Multus
-
-The selector keys are required to be `public` and `cluster` where each represent:
+If specified, the cluster network is used to isolate internal Ceph replication traffic. This includes
+additional copies of data replicated between OSDs during client reads/writes. This also includes OSD
+data recovery (re-replication) when OSDs or nodes go offline. If the cluster network is unspecified,
+the public network is used for this traffic instead.
 
-* `public`: client communications with the cluster (reads/writes)
-* `cluster`: internal Ceph replication network
+Some Rook network providers allow manually specifying the public and network interfaces that Ceph
+will use for data traffic. Use `addressRanges` to specify this. For example:
 
-If you want to learn more, please read:
-
-* [Ceph Networking reference](https://docs.ceph.com/docs/master/rados/configuration/network-config-ref/).
-* [Multus documentation](https://intel.github.io/multus-cni/doc/how-to-use.html)
+```yaml
+  network:
+    provider: host
```

Comment:
Host networking is backward compatible, we're just adding more options for public/cluster that they could also apply to host networking, right?

Reply:
Right. This doc subsection is more about what public/cluster networks are, as a primer, with the `host` option detailed below.